riscemu/decoder/__main__.py

```diff
@@ -15,5 +15,5 @@
     readline.set_completer(rlcompleter.Completer(sess_vars).complete)
     readline.parse_and_bind("tab: complete")
     code.InteractiveConsole(sess_vars).interact(
-        banner="Interaktive decoding session started...", exitmsg="Closing..."
+        banner="Interactively decoding session started...", exitmsg="Closing..."
```

Comment:
Thanks for catching the stray german word. Although the proper fixed version should probably be:
```suggestion
        banner="Interactive decoding session started...", exitmsg="Closing..."
```